runtime/src/main/java/com/flipkart/flux/util/HibernateUtil.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2012-2016, the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.flipkart.flux.util;
+
+import com.flipkart.flux.domain.*;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+/**
+ * @author shyam.akirala
+ */
+public class HibernateUtil {
```

Comment:
If you want a singleton, you can provide it using whatever DI tool we use. That way we get a complete hold on its lifecycle - which we won't if it is static.